Documentation/project-docs/developer-guide.md

```diff
@@ -298,7 +298,7 @@ Use this attribute on test methods to specify that this test may only be run on
 - `nonlinuxtests` for tests that don't run on Linux
 - `nonosxtests` for tests that don't run on OS X
 
-**[Available Test Platforms](https://github.com/dotnet/buildtools/blob/master/src/xunit.netcore.extensions/TestPlatforms.cs#L10)**
+**[Available Test Platforms](https://github.com/dotnet/buildtools/blob/master/src/Microsoft.DotNet.XUnitExtensions/TestPlatforms.cs#L10)**
```

Comment:
Should this point to arcade? Or not yet?

Reply:
done.